### ISSUE TITLE
[feature] use rekor staging to e2e tests

### ISSUE THREAD

**Issue by ianlewis** [slsa-framework/slsa-github-generator#387]
Depends on #372

**Comment by asraa** [comment]
Making this more general: use the entire Sigstore staging instance. TUF root, CT, Fulcio, and Rekor key material.

**Comment by ianlewis** [comment]
I don't think this is a blocker for the Node.js builder since we haven't had many sigstore stability issues recently.
